### DISCUSSION TITLE
How can I change the color of the front page and put all the tex in PT

### DISCUSSION THREAD

**Discussion by RodrigoCCamara** [joaomlourenco/novathesis#424] [Q&A] [answered]
Hi

I would like to change the cover from green to blue bcs its a masters degree and not a Phd. 

Also looking for to change everything to Portuguese, is it possible?

Thanks in advance

**Comment by joaomlourenco** [comment, marked as answer]
Read the README at:
https://github.com/joaomlourenco/novathesis
